webapp/javascript/util/formatDate.ts

```diff
@@ -68,7 +68,7 @@ export function dateForExportFilename(from: string, until: string) {
   const dateFormat = 'yyyy-MM-dd_HHmm';
   if (/^now-/.test(from) && until === 'now') {
     const { number, _multiplier } = convertPresetsToDate(from);
-    return `Last ${number} ${_multiplier}`;
+    return `last_${number}_${_multiplier}`;
```

Comment:
You said this function is only used in `ExportData`, right? If so can we move to that file (ExportData.tsx)? I feel like having it here is confusing. I was expecting it to be something generic that was used somewhere else.